acceptance/bundle/deployment/bind/experiment/output.txt

```diff
@@ -0,0 +1,31 @@
+
+=== Bind experiment test: 
+=== Substitute variables in the template
+=== Create a pre-defined experiment
+=== Bind experiment: Updating deployment state...
+Successfully bound databricks_mlflow_experiment with an id '[NUMID]'. Run 'bundle deploy' to deploy changes to your workspace
+
+=== Deploy bundle: Uploading bundle files to /Workspace/Users/[USERNAME]/.bundle/bind-ml-experiment-test-[UUID]/default/files...
+Deploying resources...
+Updating deployment state...
+Deployment complete!
+
+=== Read the pre-defined experiment: {
+  "name": "/Users/[USERNAME]/test-experiment[UUID]",
```

Comment:
This is not related to this PR but this highlights that experiment API does not respect /Workspace prefix in name when they are created